Makefile

```diff
@@ -1,13 +1,25 @@
-.PHONY: build
+GO_FILES?=$$(find . -name '*.go' |grep -v vendor)
 
+.PHONY: build
 build:
 	./build.sh
 
+.PHONY: build_redist
```

Comment:
Add missing .PHONY directly before the target (http://clarkgrubb.com/makefile-style-guide#phony-targets)